plugins/Marketplace/resources/qml/Packages.qml

```diff
@@ -23,6 +24,29 @@ ScrollView
 
         spacing: UM.Theme.getSize("default_margin").height
 
+        section.property: "package.sectionTitle"
+        section.criteria: ViewSection.FullString
+        section.delegate: Rectangle
+        {
+            width: packagesListview.width
+            height: sectionHeaderText.implicitHeight + UM.Theme.getSize("default_margin").height
```

Comment:
Are you sure that this should be implicitHeight and not height? Implicit height is the height that is used for re-usable components when no height was explicitly set.

Reply:
I have absolutely no idea, but you're far more experienced in qml than I so I will take it face value

Reply:
```suggestion
            height: sectionHeaderText.height + UM.Theme.getSize("default_margin").height
```